src/zarr/abc/store.py

```diff
@@ -67,6 +113,20 @@ def __exit__(
         self.close()
 
     async def _open(self) -> None:
+        """
+        Open the store.
+
+        Notes
+        -----
+        * When `mode='w'` and the store already exists, it will be cleared.
```

Comment:
```suggestion
        * When ``mode='w'`` and the store already exists, it will be cleared.
```
Double backticks for code (otherwise sphinx will try and create a link from it)